style.css

```diff
@@ -1,3 +1,7 @@
+html {
```

Comment:
- [ ] [OPTIONAL] it's best to make the website responsive by using the ```vh```/```vw``` and ```%``` instead of a fixed height and width using ```px```